pkg/scheduler/algorithm/priorities/interpod_affinity.go

```diff
@@ -106,13 +106,8 @@ func (ipa *InterPodAffinity) CalculateInterPodAffinityPriority(pod *v1.Pod, node
 	// the node.
 	pm := newPodAffinityPriorityMap(nodes)
 	allNodeNames := make([]string, 0, len(nodeNameToInfo))
-	lazyInit := hasAffinityConstraints || hasAntiAffinityConstraints
 	for name := range nodeNameToInfo {
```

Comment:
So we are saying:
We can't deduce which nodes could be counted, so create a counter for all of them. Please add this to the solution section of the PR description.

Not related to this PR: have we done the same mistake for even pod spreading?

Reply:
> We can't deduce which nodes could be counted, so create a counter for all of them.

Yes, and in the before, the overhead of using a map and initializing *int64 is too high.

> have we done the same mistake for even pod spreading?

Even pods spreading doesn't need to consider Symmetry, so it's fine.